tools/ci.sh

```diff
@@ -4,8 +4,11 @@ export RUST_BACKTRACE=1
 export CARGO_INCREMENTAL=0
 export CARGO_REGISTRIES_CRATES_IO_PROTOCOL=sparse
 
-cargo build -p lalrpop
-cargo test --all --all-features
+cargo build --bin lalrpop --features pico-args
+# build with minimal amount of features to make sure dependencies can build too.
+cargo hack build --workspace --each-feature --optional-deps
+# test with minimal amount of features plus a few extra on regex/regex-syntax
+cargo hack test --workspace --each-feature --optional-deps
```

Comment:
Just curious if there is a difference here with `--each-feature` versus using `--feature-powerset`?